# Region of Interest
roi_reduction_padding_x: 0.02
roi_reduction_padding_y: 0.01
roi_reduction_padding_z: 0.02

# Outlier Removal
use_outlier_removal: true
radius_of_outlier_removal: 0.02
